.gitignore

```diff
@@ -20,6 +20,8 @@ dump.rdb
 *.log
 *.swp
 celerybeat-schedule
+celerybeat-schedule.dir
```

Comment:
Why are these two lines added?

Reply:
Docker dev env spawns these files for me